pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
```

Comment:
Missing blank line at end of file.

Reply:
Are they mandatory? Anyway, I added a blank line at the end of the file.

Reply:
It is conventional in Python file and I tend to apply it to other file type too

Reply:
> It is conventional in Python file and I tend to apply it to other file type too

👍